src/net.h

```diff
@@ -677,6 +677,7 @@ class CNode
 
 
     uint64_t nLocalHostNonce;
+    // What local services you advertised on connection to this node
```

Comment:
@instagibbs Services are advertised to a peer in the version message, which can be sent only once. (future version messages are ignored, and are counted as misbehaviour)